README.md

```diff
@@ -1,8 +1,43 @@
-# Jarvis Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Ali Saboohi . Jarvis Consulting
 
+Hi, I'm a recent graduate from Ryerson university with a bachelor's degree in Computer Science. During my four years at Ryerson my main focus was on backend development, machine learning and computer vision where I worked on many projects such as Face Detection application using HOG features, Handwrite Recognition application and created a few websites. Currently, I am a Software and Data Engineer at Jarvis where I am trained to master the tools and frameworks necessary for professional work enviroment such as Git, Docker, Java, Spring boot, RDBMS, Maven etc. In addition to this, I have a passion for the insights data can provide. My experiences taught me how to come up with creative solutions to the real-world problems and I am looking forward to bring my experiences and skills to your team.
```

Comment:
I'm a recent graduate from Ryerson University with a bachelor's degree in Computer Science. During my four years at Ryerson, my main focus was on backend development, machine learning and computer vision, where I worked on many projects such as Face Detection application using HOG features, Handwrite Recognition application and created a few websites. Currently, I am a Software and Data Engineer at Jarvis, where I am trained to master the tools and frameworks necessary for a professional work environment such as Git, Docker, Java, Spring boot, RDBMS, Maven etc. In addition to this, I have a passion for the insights data can provide. My experiences taught me how to come up with creative solutions to real-world problems, and I am looking forward to bringing my experiences and skills to your team.